src/components/testCaseList/TestCaseList.tsx

```diff
@@ -36,6 +41,11 @@ const TestCaseList = () => {
   const [activeTab, setActiveTab] = useState<string>("passing");
   const [executeAllTestCases, setExecuteAllTestCases] =
     useState<boolean>(false);
+  const [passingCoverage, setPassingCoverage] =
```

Comment:
this is not really coverage. maybe testCaseExecutionStats? passFailStats or something else

Reply:
Sure i'll change it